src/test/java/im/mak/paddle/NodeTest.java

```diff
@@ -60,4 +62,32 @@ void errorIfCantWaitNBlocks() {
 
     }
 
+    @Nested
+    class CompileScript {
+        private final String INITIAL_SCRIPT = fromFile("wallet.ride");
+        private final String ERROR_SCRIPT = fromFile("wallet.ride").replace("i.payment", "v.payment");
+
+        @Test
+        void canCompileScript() {
+            ScriptInfo scriptInfo = node().compileScript(INITIAL_SCRIPT);
+            assertThat(scriptInfo.script()).isNotEqualTo("");
```

Comment:
What's expected? Better to add an explicit expected base64 string, which will be different for the next test of compaction

Reply:
added compiled script comparison